.circleci/config.yml

```diff
@@ -56,6 +59,9 @@ jobs:
       - install_dependencies
       - run_tests_nightly
       - upload_coverage
+      - slack/notify: &slack-fail-event
+          event: fail
+          template: basic_fail_1
```

Comment:
I'm not totally sure this works... I added some extra stuff to the end of the example here: https://support.circleci.com/hc/en-us/articles/360052728991-How-to-customize-your-Slack-messages-when-using-CircleCI-s-Slack-Orb
```
- slack/notify:
          event: fail
          custom: | 
            {
                "blocks": [
                    {
                        "type": "section",
                        "text": {
                            "type": "mrkdwn",
                            "text": "Failed: $CIRCLE_USERNAME's build of \n<$CIRCLE_BUILD_URL|$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME>\n(<https://app.circleci.com/pipelines/github/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME?branch=$CIRCLE_BRANCH|$CIRCLE_BRANCH>) \n- <$CIRCLE_BUILD_URL|$CIRCLE_JOB> failed\n\nIf this is probably a result of `algod v. indexer swagger`. See this wiki page for details:\n(<https://algorand.atlassian.net/wiki/spaces/LAMPREY/pages/2339536905/Nightly+Indexer+Tests#Q%3A-What-does-it-mean-that-the-nightly-test-failed-because-of-algod-v.-indexer-swagger%3F>)" 
                        }
                    }
                ]
            }  
```